docs/dev-guide/src/config/flags.md

```diff
@@ -267,6 +268,10 @@ When enabled, logs all SMT wrapper interaction to a file.
 
 Maximum allowed length of a log file name. If this is exceeded, the file name is truncated.
 
+## `MIN_PRUSTI_VERSION`
+
+Minimum required version of Prusti that is allowed to run. If Prusti detects that it's own version is lower than this, it will throw an error and refuse to verify files. Generally [set in a `Prusti.toml` file](providing.md#flags-2) of a crate to enforce a minimum Prusti version.
```

Comment:
```suggestion
Minimum required version of Prusti that is allowed to run. If Prusti detects that its own version is lower than this, it will throw an error and refuse to verify files. Generally [set in a `Prusti.toml` file](providing.md#flags-2) of a crate to enforce a minimum Prusti version.
```